rosdep/python.yaml

```diff
@@ -9886,6 +9886,18 @@ python3-streamlit-pip:
   '*':
     pip:
       packages: [streamlit]
+python3-streamz:
+  debian:
+    bookworm: [python3-streamz]
+    bullseye: [python3-streamz]
+  ubuntu:
```

Comment:
```suggestion
  opensuse: [python3-streamz]
  ubuntu:
```

Reply:
`python3-streamz` is only available as experimental package if I understand correctly. Moreover, on Tumbleweed it is available as `python-streamz`.